src/fsharp/service/ItemKey.fs

```diff
@@ -228,71 +222,67 @@ and [<Sealed>] ItemKeyStoreBuilder() =
                 writeILType x)
             writeILType mref.ReturnType
 
-    let rec writeType (ty: TType) =
-        match ty with
+    let rec writeType isStandalone (ty: TType) =
```

Comment:
What does `isStandalone` refer to?

Reply:
Referring to if a type is not nested as part of a symbol.

```
List<int> // when writing the key for `List<_>`, we say that type argument 'int' is not standalone
               // when writing the key for 'int', we say the type is standalone
```